src/apps/authentication/views.py

```diff
@@ -14,3 +18,31 @@ def post(self, request, *args, **kwargs):
             serializer.save()
             return Response(serializer.data, status=status.HTTP_201_CREATED)
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+class LoginView(APIView):
+    serializer_class = UserLoginSerializer
+
+    def post(self, request):
+        serializer = UserLoginSerializer(data=request.data)
+
+        if serializer.is_valid():
+            # The serializer's validate method has been executed successfully
+            user = authenticate(request, email=serializer.validated_data["email"], password=serializer.validated_data["password"])
```

Comment:
```suggestion
            user = authenticate(request, email=serializer.validated_data["email"], password=serializer.validated_data["password"])
```
I think it's better to move to Serialier layer